core/preferences/src/main/proto/settings.proto

```diff
@@ -298,6 +298,7 @@ message Settings {
     bool bottom_search_bar = 2;
     bool reverse_search_results = 3;
     bool fixed_search_bar = 4;
+    bool fixed_rotation = 5;
```

Comment:
So I had a look on how I could add this programmatically in the layout settings. I've found that a lot of settings-methods are generated; And looking in the commit-log it seemed to me that I had to add this line for these methods to be generated. This somehow is not working though. What am I missing?

Reply:
They are generated at build time, either run the project or press the "build project" button in Android Studio should fix it.

Reply:
Ah, I had to remove my calls to those non-existing methods and build it for it to generate, thanks 👍🏼